src/main/java/appeng/items/misc/ItemEncodedPattern.java

```diff
@@ -187,30 +204,49 @@ public ItemStack getOutput(final ItemStack item) {
     }
 
     private boolean addInformation(final EntityPlayer player, final IAEItemStack[] items, final List<String> lines,
-            final String label, final String and, final boolean displayMoreInfo) {
+            String label, final boolean displayMoreInfo, EnumChatFormatting color) {
         final ItemStack unknownItem = new ItemStack(Blocks.fire);
         boolean recipeIsBroken = false;
         boolean first = true;
+        List<IAEItemStack> itemsList = Arrays.asList(items);
+        List<IAEItemStack> sortedItems = itemsList.stream()
+                .sorted(Comparator.comparingLong(IAEItemStack::getStackSize).reversed()).collect(Collectors.toList());
+        boolean isFluid = false;
 
-        for (final IAEItemStack item : items) {
+        for (final IAEItemStack item : sortedItems) {
 
             if (!recipeIsBroken && item.equals(unknownItem)) {
                 recipeIsBroken = true;
             }
 
-            lines.add(
-                    (first ? label : and) + NumberFormat.getNumberInstance(Locale.US).format(item.getStackSize())
-                            + " "
-                            + Platform.getItemDisplayName(item));
-
-            if (GuiScreen.isShiftKeyDown()) {
-                final List l = item.getItemStack().getTooltip(player, displayMoreInfo);
-
-                if (!l.isEmpty()) {
-                    l.remove(0);
-                }
+            if (item.getItemStack().getItem() == GameRegistry.findItem("ae2fc", "fluid_drop")) {
```

Comment:
It would be nice to cache this fluid_drop lookup instead of calling it every time